verify/verify_test.go

```diff
@@ -57,3 +60,53 @@ func TestCommitmentVerification(t *testing.T) {
 	err = v.VerifyCommitment(c, fakeData)
 	require.Error(t, err)
 }
+
+func TestCommitmentWithTooLargeBlob(t *testing.T) {
+
+	var dataRand [2000 * 32]byte
+	_, err := rand.Read(dataRand[:])
+	require.NoError(t, err)
+	data := dataRand[:]
+
+	kzgConfig := &kzg.KzgConfig{
+		G1Path:          "../resources/g1.point",
+		G2PowerOf2Path:  "../resources/g2.point.powerOf2",
```

Comment:
wondering if we should remove this value entirely as well as `g2.point`

Reply:
It is a good call out, I think we should just remove the the kzg verifier. It is intended for usage by  DA nodes

Reply:
It is going to be a breaking change. Maybe another PR